bin/sl-deploy.js

```diff
@@ -78,11 +85,10 @@ var packageInfo = getPackageInfo(workingDir);
 serviceName = serviceName || (packageInfo ? packageInfo.name : null);
 
 if (!serviceName) {
-  console.error(
-    'Unable to detect service name, package.json has no "name" property.\n' +
-    'Please update your package.json or specify a service name.\n' +
-    'See `%s --help` for more details.', $0
-  );
+  glb.error('Unable to detect service name, {{package.json}} ' +
+      'has no \'name\' property.\n' +
```

Comment:
name has to be in `{{ ... }}` doesn't it, to protect it from translation? its an english-only key in the package.json